conf/development-app.ini

```diff
@@ -10,6 +10,10 @@ pyramid.debug_all: True
 pyramid.reload_templates: True
 
 h.authority: localhost
+h.bouncer_url: http://localhost:8000
+h.client_rpc_allowed_origins: http://127.0.0.1
+h.client_url: http://localhost:3001/hypothesis
+h.websocket_url: ws://127.0.0.1:5001/ws
```

Comment:
These are non-sensitive envvars that have no reason not to be set to sensible values in dev.

Reply:
Changed these all to localhost